requirements/edx/github.txt

```diff
@@ -43,7 +43,7 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 -e git+https://github.com/edx/opaque-keys.git@27dc382ea587483b1e3889a3d19cbd90b9023a06#egg=opaque-keys
 -e git+https://github.com/edx/ease.git@b67d2928a26fe497826b6ea359b9a3d0371548a7#egg=ease==0.1.3
 -e git+https://github.com/edx/i18n-tools.git@v0.1.1#egg=i18n-tools
--e git+https://github.com/edx/edx-oauth2-provider.git@0.5.1#egg=oauth2-provider
+-e git+https://github.com/edx/edx-oauth2-provider.git@0.5.2#egg=oauth2-provider
```

Comment:
Yes.  Otherwise it doesn't actually pin the package and it won't install a new version if the old one is installed.  You also don't need the `-e` if you are pinning correctly.